playbook/app/pb_kits/playbook/pb_icon/docs/_icon_color.html.erb

```diff
@@ -0,0 +1,7 @@
+<%= pb_rails("flex", props: {orientation: "column"}) do %>
+    <%= pb_rails("icon", props: { icon: "user", fixed_width: true, color: "primary", padding_bottom: "sm", size: "2x" }) %>
+    <%= pb_rails("icon", props: { icon: "recycle", fixed_width: true, color: "info", padding_bottom: "sm", size: "2x" }) %>
+    <%= pb_rails("icon", props: { icon: "frog", fixed_width: true, color: "data_4", padding_bottom: "sm", size: "2x" }) %>
```

Comment:
it would be best to use icons that are in the playbook-icon repo. Sorry froggy :c

Reply:
but playbook icons might be getting a frog icon soon will 100% add this back

Reply:
Agreed! Thanks for flagging it here - going to address Jason's recommendations now and will remove the two font awesome icons from the doc example (made sense to have them for testing purposes but since we are removing it soon they probably shouldn't make the final cut). Froggy4ever.